content/events/summit-brazil-24/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "[Upcoming Event] Espressif Summit Brazil 2024"
+date: 2024-06-10
+draft: false
+description: "Espressif Systems Summit in Brazil"
+tags: ["event", "announcement"]
+showDate : false
+---
+
+## Espressif Summit Brazil 2024
+
+We are pleased to announce the first in-person Espressif event in Brazil, taking place in Campinas from July 17th to 18th, from 9 am to 6 pm. On the first day, we will present talks about our most recent products, including the ESP32-P4, ESP32-C5, ESP-IDF, Nuttx, Zephyr, low-power applications, Arduino, MQTT, and more.
+
+On the second day, workshops will be presented in two different tracks, including "Introduction to ESP-IDF with the ESP32-C6," "IoT Workshop with RainMaker and Matter," and "Zephyr with ESP32." All the talks and workshops will be presented by Espressif experts and will be mostly in **Portuguese**.
+
+### Location
+
+Vitoria Hotel Concept, Campinas, SP
+
+Time: Starting from 9 AM
+
+Use the promotional code ESPRESSIF and get a 12% discount on your stay.
+
+## Agenda
+
+| Time  | Activity | Speaker |
+|-------|-----------|---------|
+| 9:00  | Credenciamento e welcome coffee | Espressif |
+| 9:30  | Abertura e keynote | Ricardo Tafas - Software Manager, Pedro Minatel - DevRel Manager |
+| 10:00 | Apresentação dos novos produtos e soluções Espressif | Pedro Minatel - DevRel Manager |
+| 11:00 | Introdução ao desenvolvimento com o ESP-IDF v5.3 | Ivan Grokhotkov - VP Platforms |
+| 11:30 | Ferramentas de desenvolvimento Espressif | Ivan Grokhotkov - VP Platforms |
+| 12:00 | Almoço (não incluso no evento) | |
+| 13:00 | Aplicações de baixo consumo com o ESP32-C6 e WiFi6 | Euripedes Rocha - Networking Manager |
+| 13:30 | Aplicações IHM e os novos recursos do ESP32-P4 | Pedro Minatel - DevRel Manager |
+| 14:00 | Introdução ao NuttX | Tiago Medicci - Software Engineer |
+| 15:00 | Coffee Break | |
+| 15:30 | Desenvolvimento Arduino como componente do ESP-IDF | Rodrigo Garcia - Software Engineer |
+| 16:00 | Boas práticas de rede e MQTT com o ESP32 | Euripedes Rocha - Networking Manager |
+| 17:00 | Introdução ao Zephyr e debugging com openOCD | Lucas Tamborrino - Software Engineer |
+| 18:00 | Q&A geral e networking | Espressif Systems |
+
+## Registration
+
+The event is free for any participant, and registration can be done for each day and workshop with the following links:
+
+- Talks on July 17th: [Espressif Summit Brazil 2024](https://espressif.us15.list-manage.com/track/click?u=40830afd8eb6f70ab5e47b7a4&id=1988e2d518&e=e3a0757cf1)
+
+Workshops July 18th:
+
+- Workshop: [Introdução ao ESP-IDF com o ESP32-C6 (EN) - 10:00 – 13:00](https://espressif.us15.list-manage.com/track/click?u=40830afd8eb6f70ab5e47b7a4&id=befed46386&e=e3a0757cf1)
```

Comment:
Suggest using the time format consistently: either 24-hour format or am/pm everywhere.